src/Products/Five/utilities/browser/marker.py

```diff
@@ -0,0 +1,61 @@
+##############################################################################
+#
+# Copyright (c) 2004, 2005 Zope Foundation and Contributors.
+# All Rights Reserved.
+#
+# This software is subject to the provisions of the Zope Public License,
+# Version 2.1 (ZPL).  A copy of the ZPL should accompany this distribution.
+# THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND ALL EXPRESS OR IMPLIED
+# WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT, AND FITNESS
+# FOR A PARTICULAR PURPOSE.
+#
+##############################################################################
+"""Marker interfaces adapter views.
+"""
+
+from Products.Five.utilities.interfaces import IMarkerInterfaces
+
+
+class EditView(object):
+    """Marker interface edit view.
+    """
+
+    def __init__(self, context, request):
+        self.context = context
+        self.request = request
+        self.adapted = IMarkerInterfaces(context)
+        self.context_url = self.context.absolute_url()
+
+    def __call__(self, SAVE=None, add=(), remove=()):
+        if SAVE:
+            self.update(add, remove)
+            self.request.response.redirect(self.request.ACTUAL_URL)
```

Comment:
Setting a `manage_tabs_message` here would be nice to hint the user that his save request was successful.

Reply:
You're right, I added that now.